gamestonk_terminal/econ/econ_controller.py

```diff
@@ -112,36 +118,43 @@ def call_fred_api(self, other_args: List[str], cmd: str):
         """Process Fred Data call"""
         fred_view.get_fred_data(other_args, cmd)
 
+    def call_VIX(self, other_args: List[str], cmd: str):
```

Comment:
and then here handle view command.

Otherwise if we want to add functions in here is just gonna be a cascade of if/elif which isn't pretty!